instrumentation/spring/README.md

```diff
@@ -592,7 +592,396 @@ To create a sample trace enter `localhost:8080/message` in a browser. This trace
+#### Application Configurations
+
+The following tracer configurations can be used to customize your instrumentation. Add the following values to your project's resource/application.properties file:
+
+```properties
+
+## TimeService will run on port 8080
+## Setting the server port of MainService to 8081 will prevent conflicts
+server.port=8081
+
+## Set Tracer name
+opentelemetry.trace.tracer.name=time_service
+opentelemetry.trace.tracer.samplerProbability=1
+
+## Default configurations
+#opentelemetry.trace.web.enabled = True
+#opentelemetry.trace.httpclients.enabled = True
+#opentelemetry.trace.tracer.samplingProbablity=1
+#opentelemetry.trace.exporter.loggin.enabled=True
+#opentelemetry.trace.aspects.enabled=True
+
+```
+
+Check out [OpenTelemetry Spring Boot AutoConfigure](spring-boot-autoconfigure/README.md) to learn more.
+
+
+
+### TimeService
+
+Configure the main class in your `Time Service` project to match the file below. Here we use the Tracer bean provided by the OpenTelemetry to create an internal span and set some additional events and attributes. 
+
+```java 
+
+import java.io.IOException;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestTemplate;
+
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.extensions.auto.annotations.WithSpan;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Tracer;
+
+@SpringBootApplication
+public class TimeServiceApplication {
+
+  public static void main(String[] args) throws IOException {
+    SpringApplication.run(TimeServiceApplication.class, args);
+  }
+
+  @RestController
+  @RequestMapping(value = "/time")
+  public class TimeServiceController {
+     @Autowired
+     private Tracer tracer;
+
+     @GetMapping
+     public String time() {
+        withSpanMethod();
+
+        Span span = tracer.spanBuilder("time").startSpan();
+        try (Scope scope = tracer.withSpan(span)) {
+           span.addEvent("TimeServiceController Entered");
+           span.setAttribute("what.am.i", "Tu es une legume");
+           return "It's time to get a watch";
+        } finally {
+           span.end();
+        }
+     }
+     
+     @WithSpan(kind=Span.Kind.SERVER)
+     public void withSpanMethod() {}
+  }
+}
+```
+
+
+### Generating Trace - LoggingSpanExporter
+
+To generate a trace run MainServiceApplication and TimeServiceApplication and then send a request to `localhost:8080/message`. Shown below is the output of the the default span exporter - (LoggingSpanExporter)[https://github.com/open-telemetry/opentelemetry-java/tree/master/exporters/logging].
```

Comment:
is LoggingSpanExporter the default, or does user need to add dependency for it too?
```suggestion
To generate a trace, run MainServiceApplication and TimeServiceApplication, and then send a request to `localhost:8080/message`. Shown below is the output of the default span exporter - (LoggingSpanExporter)[https://github.com/open-telemetry/opentelemetry-java/tree/master/exporters/logging].
```

Reply:
LoggingSpanExporter is the default exporter provided by the opentelemetry-spring-starter. Users won't need to add this dependency.